### PR TITLE
Revert "Include XKit version as a header in requests to Tumblr"

### PR DESCRIPTION
**Pull request by nightpool**
Reverts new-xkit/XKit#1744

On some chrome installs (but not others?) this triggers a CORS preflight check. This didn't show up in testing, and it's really grinding my gears. Sorry @cyle

### CODE REVIEW

**Comment by cyle** [comment]
wow... that doesn't seem likely to me... you're already setting custom headers on every request, this is just an additional one. maybe it's what you did to wrap the `GM_xmlhttpRequest` stuff? or maybe we could retry this without that work.

**Comment by cyle** [comment]
ah i see from [here](https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request) and [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Simple_requests) -- probably adding them to the `$.ajax()` calls did it

**Comment by cyle** [comment]
i'd approve this PR if i could 😄

**Comment by nightpool** [comment]
it's not ready yet—I need to bump the versions and add code to force-update
all of the extensions before patches can load and clobber GM_XMLHttpRequest

File: Extensions/xkit_patches.js [hunk 1]
```diff
@@ -161,17 +161,10 @@ XKit.extensions.xkit_patches = new Object({
 		}, 1000);
 	},
 
-	run_order: ["7.8.1", "7.8.2", "7.9.0"],
+	run_order: ["7.8.1", "7.8.2", "7.9.1"],
```
Comment (AprilSylph) [inline Extensions/xkit_patches.js]: 7.9.0 needs to be in there so that it doesn't run 7.8.x patches

Comment (AprilSylph) [reply Extensions/xkit_patches.js]: i guess this means we need an empty 7.9.0 function too. sorry that my patches structure isnt nice for this type of core patch :sweat:

**Comment by nightpool** [comment]
No worries! that's an easy enough change

**Pull request merged**